Map/Test/MapTest.cpp

```diff
@@ -155,8 +142,52 @@ TEST_F(SquareMapTest, treasureFieldInvisiblePrint)
   ASSERT_EQ(' ',map.printField(field));
 }
 
-TEST_F(SquareMapTest, wallFieldInvisiblePrint)
+TEST_F(SquareMapTest, wallFieldVisiblePrint)
 {
-  WallField field;
-  ASSERT_EQ(' ',map.printField(field));
+    WallField field;
+    ASSERT_EQ('#', map.printField(field));
+}
+
+TEST_F(SquareMapTest, doorFieldVisiblePrint)
+{
+    DoorField field;
+    ASSERT_EQ('D', map.printField(field));
+}
+
+// Visilibity Updater
+
+TEST_F(SquareMapTest, roomShouldBeInvisibleByDefault)
+{
+    SquareMap map(mapSize);
```

Comment:
mapa juz jest stworzona w klasie SqureMapTest, mozna jej reużyć zamist tworzyć nową. Chyba że koniecznie potrzebujesz tu innej mapy

Reply:
Ten test działa, bo aktualnie przy tworzeniu mapy nie tworza sie ani drzwi, ani sciany ;) Jakby byly drzwi to wydaje mi sie, że bylby od razu widoczne przez kod który napisales wyżej

Reply:
> Ten test działa, bo aktualnie przy tworzeniu mapy nie tworza sie ani drzwi, ani sciany ;)Jakby byly drzwi to wydaje mi sie, że bylby od razu widoczne przez kod który napisales wyżej

Tak

Reply:
To się okaże po konsultacji z grupą na slacku :)